### ISSUE TITLE
`Open in browser` tab is not working for presto/trino connector

### ISSUE THREAD

**Issue by AbhiY-6sense** [cloudera/hue#3200]
Is the issue already present in https://github.com/cloudera/hue/issues or discussed in the forum https://discourse.gethue.com?

`No`

Describe the bug:

```
{'message': 'line 2:6: backquoted identifiers are not supported; use double quotes to quote identifiers', 'errorCode': 1, 'errorName': 'SYNTAX_ERROR', 'errorType': 'USER_ERROR', 'errorLocation': {'lineNumber': 2, 'columnNumber': 6}, 'failureInfo': {'type': 'io.trino.sql.parser.ParsingException', 'message': 'line 2:6: backquoted identifiers are not supported; use double quotes to quote identifiers', 'suppressed': [], 'stack': ['io.trino.sql.parser.SqlParser$PostProcessor.exitBackQuotedIdentifier(SqlParser.java:192)', 'io.trino.sql.parser.SqlBaseParser$BackQuotedIdentifierContext.exitRule(SqlBaseParser.java:17228)', 'org.antlr.v4.runtime.Parser.triggerExitRuleEvent(Parser.java:408)', 'org.antlr.v4.runtime.Parser.exitRule(Parser.java:642)', 'io.trino.sql.parser.SqlBaseParser.singleStatement(SqlBaseParser.java:305)', 'io.trino.sql.parser.SqlParser.invokeParser(SqlParser.java:145)', 'io.trino.sql.parser.SqlParser.createStatement(SqlParser.java:85)', 'io.trino.execution.QueryPreparer.prepareQuery(QueryPreparer.java:55)', 'io.trino.dispatcher.DispatchManager.createQueryInternal(DispatchManager.java:180)', 'io.trino.dispatcher.DispatchManager.lambda$createQuery$0(DispatchManager.java:149)', 'io.trino.$gen.Trino_380____20230208_121044_2.run(Unknown Source)', 'java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)', 'java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)', 'java.base/java.lang.Thread.run(Thread.java:829)'], 'errorLocation': {'lineNumber': 2, 'columnNumber': 6}}}

```

Steps to reproduce it?

- Select Presto/Trino connector
- Select any database 
- Right click on any table then select `open in browser` option, you will see above error log on screen 

Hue version or source? (e.g. open source 4.5, CDH 5.16, CDP 1.0...). System info (e.g. OS, Browser...).
- Version 4.10
- OS Mac
- Browser chrome

**Comment by bjornalm** [comment]
Hi @AbhiY-6sense, thanks for reporting. @nidhibhatg can you take a look?

**Comment by Harshg999** [comment]
Hey @AbhiY-6sense, can you once try out the 4.11 release?
This commit (https://github.com/cloudera/hue/commit/bafce33ad8a0ec41299864bdbe9224c01ad82727) was part of the recently released 4.11 version.

**Comment by Harshg999** [comment]
@AbhiY-6sense - Please once try out the 4.11.0 release or the commit changes directly to see if it resolves this issue. Feel free to open this issue again if it doesn't.

Thanks!
